src/main/java/frc/robot/subsystems/CellSuperstructure.java

```diff
@@ -60,10 +56,13 @@
     /** True when intake just stopped itself */
     private boolean m_intakeDone = false;
 
+    /** logger instance */
+    private RobotLogger m_loggerInstance = RobotLogger.getInstance();
```

Comment:
For consistency, this should be named `logger` not `m_loggerInstance`

Reply:
I just renamed the variable